### PR TITLE
JPN-588 Separate out Admins from the Contribution module and emphasise 'latest admins'

### PR DESCRIPTION
**Pull request by perjg**
With this change, the admin module is separated out from the main contributions module, and the query is modified to return last revision date, ordered by last revision date.

@Wikia/spitfires @d34th4ck3r

### CODE REVIEW

**Comment by jkuner** [comment]
@Wikia/spitfires can you please provide a code review? Thank you!

File: extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php [hunk 1]
```diff
@@ -117,7 +117,7 @@ function () {
 				$dateTwoYearsAgo = date( 'Y-m-d', strtotime( '-2 years' ) );
 
 				$sqlData = ( new WikiaSQL() )
-					->SELECT( 'rev_user_text, rev_user, wup_value' )
+					->SELECT( 'rev_user_text, rev_user, wup_value, MAX(rev_timestamp) AS latest_revision' )
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php]: Do you still need `wup_value` if you replaced # of contributions with last revision date?

Comment (perjg) [inline extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php]: Good catch... I don't need it any more, will take it out.

**Comment by perjg** [comment]
@lukaszkonieczny Updated the PR according to your comment.

File: extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php [hunk 2]
```diff
@@ -127,12 +127,12 @@ function () {
 					->AND_( 'rev_timestamp' )->GREATER_THAN( $dateTwoYearsAgo )
 					->AND_( 'wup_property' )->EQUAL_TO( 'editcount' )
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php]: If we removed `wup_value` I think we can also remove `wup_property` and whole `LEFT JOIN`

**Comment by perjg** [comment]
@lukaszkonieczny Optimized the query too, sorry for missing that.

**Comment by lukaszkonieczny** [comment]
Thanks :+1: LGTM

**Comment by perjg** [comment]
@lukaszkonieczny One more commit to add legend to the top of the all admins list in the modal tab.

**Comment by lukaszkonieczny** [comment]
LGTM

**Comment by d34th4ck3r** [comment]
LGTM for last commit.

**Pull request merged**